homeassistant/components/youtube/sensor.py

```diff
@@ -84,3 +90,10 @@ def native_value(self) -> StateType:
     def entity_picture(self) -> str:
         """Return the value reported by the sensor."""
         return self.entity_description.entity_picture_fn(self._channel)
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any] | None:
+        """Return the extra state attributes."""
+        if self.entity_description.attributes_fn:
+            return self.entity_description.attributes_fn(self._channel)
+        return None
```

Comment:
The attribute keys should be consistent, right now, they are not.

Please make sure they keys of the dict are present consistently.

../Frenck